src/test/suite/participant/participant.test.ts

```diff
@@ -1838,6 +1839,10 @@ Schema:
             {
               message: `I want to ask questions about the \`${mockDatabaseItem.databaseName}\` database.`,
               isNewChat: true,
+              telemetry: {
+                source: DocumentSource.DOCUMENT_SOURCE_TREEVIEW,
+                sourceDetails: 'copilot button on database tree item',
```

Comment:
Can it be a case with one source but different sourceDetails? Maybe we could pass only one arg something like telemetrySource and then match them somewhere in one place to sourceDetails message?

Reply:
I was thinking of having `telemetry` be an object in case in other cases (i.e. in other functions) we'd like to include more properties to include in tracking. Then we could have somewhat of a consistent structure when we pass telemetry information to functions / commands we use.

Could you clarify what you mean by matching in one place to sourceDetails? Not sure I follow

Reply:
What is the telemetry event that is sent in this case?

Reply:
```
  PARTICIPANT_INPUT_BOX_SUBMITTED = 'Participant Inbox Box Submitted'
```

Reply:
Yeah, so my question is, why do we need textual `sourceDetails`, when we already have `source`? Like based on `source=DocumentSource.DOCUMENT_SOURCE_TREEVIEW` we know this was triggered from the tree view. And based on 'Participant Inbox Box Submitted', we know this is from copilot. Do we really need to send `copilot button on database tree item`?

Reply:
that's fair. I was just thinking whether we'd care about whether users care from a collection item or a database item instead of tree view in general. `DocumentSource` so far has been kind of high-level so far i.e. 
```ts
export enum DocumentSource {
  DOCUMENT_SOURCE_TREEVIEW = 'treeview',
  DOCUMENT_SOURCE_PLAYGROUND = 'playground',
  DOCUMENT_SOURCE_COLLECTIONVIEW = 'collectionview',
  DOCUMENT_SOURCE_CODELENS = 'codelens',
}
```
I could either just not care about details or add new categories if we do care about tracking which type of it this is.

Reply:
applied this; maybe still `source_details` or `source_info`? `source_type` seems too similar to `source` and its hard to differentiate

Reply:
Maybe then something very explicit like `source_tree_item`? `source_details` and `source_info` make me expect some kind of object or a long description.

Reply:
But it can be no a tree next time... Ok I agree on as it is now with source_details :)